chromium_src/components/password_manager/core/browser/password_store/password_store_built_in_backend.cc

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "build/build_config.h"
+
+#if BUILDFLAG(IS_ANDROID)
+
+// Without this override there are error on Android cr130 build:
+// password_store_built_in_backend.cc:217:36: error: use of undeclared
+// identifier 'features'; did you mean 'sql::features'?
+//   217 |       base::FeatureList::IsEnabled(features::kUseNewEncryptionMethod)
+//       |                                    ^~~~~~~~
+//       |                                    sql::features
+// ../../sql/sql_features.h:11:16: note: 'sql::features' declared here
+//    11 | namespace sql::features {
+//       |                ^
+// password_store_built_in_backend.cc:217:46: error: no member named
+// 'kUseNewEncryptionMethod' in namespace 'sql::features'
+//   217 |       base::FeatureList::IsEnabled(features::kUseNewEncryptionMethod)
+//       |                                    ~~~~~~~~~~^
+//
+// The mentioned enums are at
+// "components/password_manager/core/browser/features/password_features.h",
+// which is included under pre-processor guards: #if
+// !BUILDFLAG(USE_LOGIN_DATABASE_AS_BACKEND) #include
+// "components/password_manager/core/browser/features/password_features.h"
+// #endif  // !BUILDFLAG(USE_LOGIN_DATABASE_AS_BACKEND)
+// #if !BUILDFLAG(IS_ANDROID)
+// #include
+// "components/password_manager/core/browser/features/password_features.h"
+// #endif  // !BUILDFLAG(IS_ANDROID)
+// Brave set `args.use_login_database_as_backend = true` to use Chromium's
```

Comment:
maybe you can upstream a proper fix? Chromium code kinda assumes that it should be buildable with the flag set to false

Reply:
yes, I'll  build the clean Chromium with `use_login_database_as_backend = true`, if it will fail [I am sure it must], then I'll try to push fix to upstream

the problem is they are going to drop support of login database at Android in November, so there is no much sense for them.

Reply:
ah i see they plan to drop the flag in november https://source.chromium.org/chromium/chromium/src/+/main:components/password_manager/core/browser/buildflags.gni;l=11?q=use_login_database_as_backend&ss=chromium&start=21

Reply:
> so there is no much sense for them.
yeah i agree, sorry for not looking at the flag beforehand